week_1/starter/src/components/TodoItem.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+class TodoItem extends React.Component {
+  render() {
+    const {id, text, completed, priority} = this.props.todo;
+    return (
+      <li className='todo-item-container'>
+        <span style={{textDecoration: completed?"line-through":"none",}}>
+          <div style={{
```

Comment:
LGTM 🔥 One little thing you can improve is to refactor this part into a separate component **_Priority_** and pass the priority as a prop to it 
```javascript 
<Priority priority={priority}/>
```

Reply:
thanks for your help i'll do it